tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -15,7 +15,7 @@ script_dir="$(dirname $(readlink -f $0))"
 AGENT_VERSION=${AGENT_VERSION:-}
 GO_AGENT_PKG=${GO_AGENT_PKG:-github.com/kata-containers/agent}
 RUST_AGENT_PKG=${RUST_AGENT_PKG:-github.com/kata-containers/kata-containers}
-RUST_AGENT=${RUST_AGENT:-no}
+RUST_AGENT=${RUST_AGENT:-yes}
```

Comment:
Hi @lining2020 , thanks for your PR. Two questions:
- you have modified the called parts, how about the caller part ( where set the `RUST_AGENT=yes`)? If the default value is set to yes, the caller can need not set it explicitly anymore.
- In Kata containers 2.0, there will be only rust agent.

Reply:
Hi @liubin ，thanks for reviewing my PR.  To your questions:

- I found that the caller part all did not set the `RUST_AGENT=yes` in osbuilder.
```
[root@centos8 kata-containers]# grep -rn ROOTFS_BUILDER tools/osbuilder/
tools/osbuilder/Makefile:8:ROOTFS_BUILDER := $(MK_DIR)/rootfs-builder/rootfs.sh
tools/osbuilder/Makefile:86:    $(call silent_run,Creating rootfs for "$*",$(ROOTFS_BUILDER) -o $(VERSION_COMMIT) -r $(ROOTFS_BUILD_DEST)/$*_rootfs $*)
tools/osbuilder/Makefile:145:   $(ROOTFS_BUILDER) -o $(VERSION_COMMIT) -r $@
tools/osbuilder/Makefile:163:   @ $(ROOTFS_BUILDER) -l
```
- Maybe I can subbmit a PR v2 to remove all the GO AGENT logic in osbuilder. But it seems to introduce problem that some OS image releases like alpine will not supported anymore until they support rust version agent.

Reply:
And there is already a PR is doing the same thing https://github.com/kata-containers/kata-containers/pull/418/files#diff-9be820d4d1eab4601e2ce130d036eb7cL18

Reply:
OK, I see.